docs/analyzer/user_guide.md

```diff
@@ -645,6 +645,7 @@ usage: CodeChecker analyze [-h] [-j JOBS]
                            [--tidyargs TIDY_ARGS_CFG_FILE]
                            [--tidy-config TIDY_CONFIG] [--timeout TIMEOUT]
                            [--ctu | --ctu-collect | --ctu-analyze]
+                           [--ctu-ast-creation {from-pch, parsed-on-demand}]
```

Comment:
CodeChecker check command also has this option. Please extend the documentation for the check command too.